doc/reference/src/main/asciidoc/configuration.asciidoc

```diff
@@ -93,43 +90,87 @@ You can configure the CDI TCK to just run tests related to the CDI Lite specific
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
-                <excludedGroups>cdi-full,integration,javaee-full,se</excludedGroups>
+              <excludedGroups>cdi-full,integration,javaee-full,se</excludedGroups>
```

Comment:
Since you only scan the `jakarta.enterprise:cdi-tck-core-impl` dependency, there should be no `integration` and `javaee-full` tests. Excluding just `cdi-full` and `se` should be enough.

Reply:
Actually it seems there are some `integration` tests in the `cdi-tck-core-impl` module... I'll need to look at it properly next week.

Reply:
The org.jboss.cdi.tck.tests.event.observer.ObserverNotificationTest is marked as INTEGRATION, but this seems to only test features within the scope of Lite. Same with ConditionalObserverTest#testConditionalObserverMethodNotInvokedIfNoActiveContext. Yes, javaee-full should never appear in the cdi-tck-core-impl artifact.

Reply:
`cdi-tck-core-impl` module doesn't seem to have any tests with `javaee-full` group so we can perhaps remove any mention of that group from here @starksm64?

The `integration` group is technically ok to be present in `cdi-tck-core-impl` since that group states:

> Integration test - cannot run in embedded arquillian container

I take it that the embedded container is referring to [Weld's Arq. container](https://github.com/arquillian/arquillian-container-weld) which has some limitations due to its implementation (such as merging beans.xml or automatically activating contexts for test duration).
So any other CDI implementation, be it Lite of Full, can theoretically execute these tests fine (just like Weld can while running WFLY for instance).